api/reporting/schemas/REPORTING-SCHEMA-TOOLS.md

```diff
@@ -0,0 +1,177 @@
+# Reporting Schema Tools & Process
+This directory and subdirectories archive schema changes
+and tools to generate and translate them into code. There
+are a few different stages, but the grunt work is mostly automated
+for repeatability and consistency.
+
+The ruby script `generate_all_tables.rb` will operate on all of the input CSV files (described below)
+and produce output files in this directory structure 
+```
+$ ls ~/scratch/reportingOut5
+big_query_json       projection_interface projection_query     swagger_yaml
+```
+
+## EXPLAIN Application DB Table
+The `explain table` MySql [statement](https://dev.mysql.com/doc/refman/8.0/en/explain.html) generates a tabular
+set of attributes for table. These are then saved as CSV files and placed into the `mysql_describe_csv` directory.
```

Comment:
Is the csv generation manual or is there a ruby script that does this? Do the csv file names have to match the table name?

Reply:
It's manual at the moment. Yes, it needs to match the table name.